R/save_load.R

```diff
@@ -0,0 +1,45 @@
+#' Save models to disk and load models from disk
+#'
+#' @description Note that model objects contain training data, except columns
+#'   ignored (patient_id in the example below). Therefore, if there is PHI in
+#'   the training data, the saved model object must be treated as PHI.
+#'   \code{save_models} issues a message saying as much.
+#'
+#' @param x model_list object
+#' @param filename File path to save model to or read model from, e.g.
+#'   "models/my_models.RDS". Default for \code{save_models} is "models.RDS" in
+#'   the working directory (\code{getwd()}). Default for \code{load_models} is
+#'   to open a dialog box from which a file can be selected, in which case a
+#'   message will issued with code to load the same file without interactivity.
+#' @return \code{load_models} returns the model_list which can be assigned to
+#'   any variable name
+#'
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' m <- machine_learn(pima_diabetes, patient_id, outcome = diabetes)
+#' save_models(m, "diabetes_models.RDS")
+#' # Restart R, move RDS file to another computer, etc.
+#' m2 <- load_models("diabetes_models.RDS")
+#' all.equal(m, m2)
+#' }
+save_models <- function(x, filename = "models.RDS") {
+  saveRDS(x, filename)
+  message("The model object being saved contains training data, minus ignored ID columns.\n",
+          "If there was PHI in training data, normal PHI protocols apply to the RDA file.")
+  return(invisible(NULL))
```

Comment:
Thanks for adding this so quickly! Perfect way to not only warn about PHI but also make saving more intuitive for newbies.

There's an `RDA` mentioned here. Do we want to standardize to `RDS`?

Reply:
Thanks @levithatcher. Yes, all the RDAs should've been replaced with RDS. I will fix. 

FYI, I got rid of the interactive option for saving. It's still there for loading, which I think is the more important one. For save, it defaults to `models.RDS` in the working directory. I dropped it because it introduced a dependency on something for X11 quartz that would've required an apt install on Travis and I was worried about compatibility across all systems. So we lost the cool 1994 dialog box look. :(